frontend/web/src/app/services/order/order.service.ts

```diff
@@ -0,0 +1,64 @@
+import { Injectable } from '@angular/core';
+import {Headers, Http, Response, ResponseOptions} from '@angular/http';
+import {Observable} from 'rxjs';
+import {Order} from '../../model/order';
+import {HttpClientService} from '../http/http-client.service';
+import {HttpClient} from '../http/http.client';
+
+@Injectable()
+export class OrderService {
+  private endpoint: string = HttpClient.orderUrl;
+
+  constructor(private http: Http, private client: HttpClientService) {}
+
+  getOrder(id: number): Observable<Order> {
+    return this.client.get(this.endpoint + '/' + id)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use a template literal instead of concatenating with a string literal. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=typescript%3AS3512)